server/lib/auth.js

```diff
@@ -93,16 +95,56 @@ export function verifyTwoFactorAuthenticatorCode(encryptedTwoFactorAuthToken, tw
   return verified;
 }
 
-export function enforceTwoFactorAuthenticationOnPayouts(req, twoFactorAuthenticatorCode) {
+const getCacheKey = userId => {
+  return `${userId}_2fa_payment_limit`;
+};
+
+export async function rollingPayoutLimitTwoFactorAuthentication(
+  req,
+  twoFactorAuthenticatorCode,
+  hostRollingLimit,
+  expenseAmount,
+) {
   if (req.remoteUser.twoFactorAuthToken !== null) {
+    let cacheObject;
```

Comment:
If we're returning this object on every conditional, we don't need to flag `cacheObject`.